### PR TITLE
Added initial integration for messenger rooms

### PR DESCRIPTION
**Pull request by ekohilas**

### CODE REVIEW

**Comment by ekohilas** [comment]
@madsmtm can this be pulled? It fixes the errors when getting a list of chat threads that contain a room

**Comment by madsmtm** [comment]
I'm sorry, haven't had the time to look through it, but I've done so now. Looks good! ;)

File: fbchat/graphql.py [hunk 1]
```diff
@@ -118,6 +118,16 @@ def graphql_to_group(group):
         message_count=group.get('messages_count')
     )
 
+def graphql_to_room(room):
+    return Room(
+        **vars(graphql_to_group(room)),
```
Comment (madsmtm) [inline fbchat/graphql.py]: This line is failing:
```
Traceback (most recent call last):
  File "tests.py", line 12, in <module>
    from fbchat import Client
  File "./fbchat/__init__.py", line 5, in <module>
    from .client import *
  File "./fbchat/client.py", line 13, in <module>
    from .graphql import *
  File "./fbchat/graphql.py", line 123
    **vars(graphql_to_group(room)),
                                  ^
SyntaxError: invalid syntax
```
In python 2.7 and python 3.4

Comment (madsmtm) [reply fbchat/graphql.py]: I reverted the commit because of this, but somehow I still closed the PR

Comment (ekohilas) [reply fbchat/graphql.py]: And I don't know how to re-open it :p 
I've made a new PR with the given fix

**Pull request merged**